packages/pipeline/test/entities/copper_test.ts

```diff
@@ -0,0 +1,58 @@
+import 'mocha';
+import 'reflect-metadata';
+
+import {
+    CopperActivity,
+    CopperActivityType,
+    CopperCustomField,
+    CopperLead,
+    CopperOpportunity,
+} from '../../src/entities';
+import { createDbConnectionOnceAsync } from '../db_setup';
+import * as activityTypesData from '../fixtures/copper/api_v1_activity_types.parsed.json';
+import * as customFieldsData from '../fixtures/copper/api_v1_custom_field_definitions.parsed.json';
+import * as activitiesData from '../fixtures/copper/api_v1_list_activities.parsed.json';
+import * as leadsData from '../fixtures/copper/api_v1_list_leads.parsed.json';
+import * as opportunitiesData from '../fixtures/copper/api_v1_list_opportunities.parsed.json';
+import { chaiSetup } from '../utils/chai_setup';
+
+import { testSaveAndFindEntityAsync } from './util';
+
+chaiSetup.configure();
+
+// tslint:disable:custom-no-magic-numbers
+describe('Copper entities', () => {
+    describe('save and find', async () => {
+        it('Copper lead', async () => {
+            const connection = await createDbConnectionOnceAsync();
+            const repository = connection.getRepository(CopperLead);
+            const entity = (leadsData[0] as unknown) as CopperLead;
```

Comment:
It seems like all of the fixture data is an array but we only ever use the first element. We should either use all the elements in the array or just change the fixture data to be an object.

Reply:
Yep, we only use the first element. It's an array because the fixtures data is also used in the parsers test. I deliberately coupled the fixtures for both set of tests so that the tests also fail if the parser output stops being congruent with the schema. Does that make sense?

Reply:
I was mainly concerned about it taking too long, but I just switched it and testing all entities takes 13.25s vs 11.85s. Seems alright to me, wdyt?